apps/email/docs/startup.md

```diff
@@ -0,0 +1,63 @@
+## Email startup flow
```

Comment:
This doc is an awesome addition!  A partner engineer in particular was looking for some docs about the email app startup the other day, and I think they would have found this invaluable.  (And appreciated the cleaned-up flow.)  The actual request for this is partially why I'm requesting the extra comments in the implementation.

One small improvement here I think would be to briefly mention that `model` is responsible for starting up the back-end (and is synonymous with the back-end really) in the "Email startup flow" section.